### PR TITLE
Make substitution black easy to mine

### PR DESCRIPTION
**Pull request by xavierh**
- Make the substitution blocks easy to mine.
- add recipe for baker's hut
- various warnings removed
Review please

### CODE REVIEW

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 35 issues
* ![MINOR][MINOR] 33 minor
* ![INFO][INFO] 2 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [BlockSolidSubstitution.java#L51](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/coremod/blocks/BlockSolidSubstitution.java#L51): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [BlockSubstitution.java#L51](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/coremod/blocks/BlockSubstitution.java#L51): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/a43128fd540d8caa51c9ec41253811e2beb194ed/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Raycoms** [comment]
:shipit: sure

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1132/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

File: src/main/java/com/minecolonies/coremod/util/RecipeHandler.java [hunk 1]
```diff
@@ -73,6 +73,7 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
 
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutWareHouse, 1), "XXX", "X#X", "XXX", 'X', PLANK_WOOD, '#', "chest")); // check
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutDeliveryman, 1), "XXX", "X#X", "XXX", 'X', PLANK_WOOD, '#', Items.LEATHER_BOOTS));
+        GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutBaker, 1), "XXX", "X#X", "XXX", 'X', PLANK_WOOD, '#', Items.WHEAT));
 
         //enableInDevelopmentFeatures(enableInDevelopmentFeatures);
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/util/RecipeHandler.java:78]: MINOR This block of commented-out lines of code should be removed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 38 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 34 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [BuildingFarmer.java#L518](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingFarmer.java#L518): Reduce this class from 86 to the maximum allowed 30 or externalize it in a public class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2972)
1. ![MAJOR][MAJOR] [RecipeHandler.java#L82](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L82): Remove this unused private "enableInDevelopmentFeatures" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [BlockSolidSubstitution.java#L51](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/coremod/blocks/BlockSolidSubstitution.java#L51): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [BlockSubstitution.java#L51](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/coremod/blocks/BlockSubstitution.java#L51): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/4b6420489b6e50cc6e0296343df2302d8826d154/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by marvin-bitterlich** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1132/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 38 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 34 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [BuildingFarmer.java#L518](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingFarmer.java#L518): Reduce this class from 86 to the maximum allowed 30 or externalize it in a public class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2972)
1. ![MAJOR][MAJOR] [RecipeHandler.java#L82](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L82): Remove this unused private "enableInDevelopmentFeatures" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [BlockSolidSubstitution.java#L51](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/coremod/blocks/BlockSolidSubstitution.java#L51): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [BlockSubstitution.java#L51](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/coremod/blocks/BlockSubstitution.java#L51): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/fd44557b14b9070e1e159c24cd958a3a4a452a79/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**